modules/actor/actor-wfrp4e.js

```diff
@@ -105,7 +105,7 @@ export default class ActorWfrp4e extends WFRP4eDocumentMixin(Actor)
     let effects;
     if (collection == "items")
     {
-      effects = documents.reduce((effects, item) => effects.concat(item), []);
+      effects = documents.reduce((effects, item) => effects.concat(item.effects.map(x=>x)), []);
```

Comment:
`effects.map(x=>x)` does nothing. Did you mean filter?

Reply:
when i was testing, item.effects in concat was not treated as an array/iterable, instead it was applying whole objects collection which was breaking stuff.

Reply:
Then you just want `item.effects.contents`, which is an array, `item.effects` is a collection so doesn't natively concat